app/helpers/application_helper.rb

```diff
@@ -303,11 +303,11 @@ def deployed_or_running_list(stages, reference)
     stages.each do |stage|
       next unless deploy = stage.deployed_or_running_deploy
       next unless deploy.references?(reference)
+      label = (deploy.active? ? "label-warning" : "label-success")
```

Comment:
Why ‘warning’ and not e.g. ‘active’?

Reply:
warning is what we used before, it makes it yellow